superset/views/core.py

```diff
@@ -1838,7 +1838,8 @@ def testconn(self):
             engine = create_engine(uri, **engine_params)
 
             with closing(engine.connect()) as conn:
-                return json_success(json.dumps(conn.scalar(select([1]))))
+                conn.scalar(select([1]))
+                return json_success("'OK'")
```

Comment:
a different api just returns `'OK'` (https://github.com/apache/incubator-superset/blob/f7812a3f27304f6ccb5932075891f162264cd88b/superset/views/core.py#L1549). which one is correct? Can we make this one and the other one consistent?

Reply:
@etr2460 a JSON encoded string much contain double quotes:

```
>>> import json
>>> json.loads('"OK"')
'OK'
```

I've updated the example above. Note the logic is somewhat inconsistent as `json_response` expects a Python object to which is dumped to a JSON string whereas `json_success` expects a JSON encoded string.